### PR TITLE
prometheus: Add mgmt_process_start_time_seconds

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: pgraph/pgraph.go [hunk 1]
```diff
@@ -624,6 +626,9 @@ func (g *Graph) GraphMetas() []*resources.MetaParams {
 
 // AssociateData associates some data with the object in the graph in question.
 func (g *Graph) AssociateData(data *resources.Data) {
+	// prometheus needs to be associated to this graph as well
+	g.prometheus = data.Prometheus
```
Comment (purpleidea) [inline pgraph/pgraph.go]: Okay, but where is this getting used?

Comment (roidelapluie) [inline pgraph/pgraph.go]: Oops that is for the next PR.

**Comment by roidelapluie** [comment]
Updated

**Comment by roidelapluie** [comment]
updated

**Comment by purpleidea** [comment]
@roidelapluie check your patch, i think you pushed the wrong thing.

**Comment by purpleidea** [comment]
(maybe didn't rebase to new git master-- old patch is in there, and commit msg is last one)

File: resources/file.go [hunk 2]
```diff
@@ -676,6 +676,13 @@ func (obj *FileRes) chmodCheckApply(apply bool) (checkOK bool, _ error) {
 	}
 
 	mode, err := obj.mode()
+
+	// If the file does not exist and we are in
+	// noop mode, do not throw an error.
+	if os.IsNotExist(err) && !apply {
+		return false, nil
+	}
+
```
Comment (mildred) [inline resources/file.go]: good catch :)

**Comment by roidelapluie** [comment]
sorry, fixed

**Comment by roidelapluie** [comment]
On 14 Feb 13:13, mildred wrote:
> mildred commented on this pull request.
> good catch :)

Was fixed in #145 :)

-- 
 (o-    Julien Pivotto
 //\    Open-Source Consultant
 V_/_   Inuits - https://www.inuits.eu

**Pull request merged**